docs/topics/braintree.rst

```diff
@@ -261,3 +260,27 @@ Some information about the subscripton is stored in solitude.
     :query paymethod__braintree_buyer: the primary key of the braintree buyer.
     :query paymethod__braintree_buyer__buyer: the primary key of the buyer.
     :query seller_product: the primary key of the product.
+
+Webhook
+-------
+
+When Braintree completes certain actions, webhooks will be sent. For more information
+see the `Braintree documentation <https://developers.braintreepayments.com/javascript+python/reference/general/webhooks>`_.
+
+.. http:get:: /braintree/webhook/verify/
+
+    .. code-block:: json
+
+        "some-token"
+
+    :<json bt_challenge: the bt_challenge issued by Braintree.
+    :>json string: a token returned by the Braintree verify API.
+
+    :status 200: token verified and returned.
+
+.. http:post:: /braintree/webhook/parse/
+
+    :<json bt_signature: the bt_signature issued by Braintree.
```

Comment:
Which doc, the Braintree one? Again goes through payment-service which posts using curling.

https://github.com/mozilla/payments-service/pull/21/files#diff-340a6a494441afe57ca1271942b853c3R116